admin/services/data-access/check-window.data.service.js

```diff
@@ -237,6 +237,32 @@ const checkWindowDataService = {
     ]
     const result = await sqlService.query(sql, params)
     return R.head(result)
+  },
+
+  /**
+   * Find all check windows
+   * @param sortBy
+   * @param sortDirection
+   * @return {Object}
```

Comment:
This function returns an array of object?  So it should be `@returns [{Object}] - array of checkWindows`

Reply:
@jon-shipley correct, it's updated